core/image/astc_format.go

```diff
@@ -42,3 +45,44 @@ func (f *FmtASTC) check(data []byte, w, h, d int) error {
 func (*FmtASTC) channels() stream.Channels {
 	return stream.Channels{stream.Channel_Red, stream.Channel_Green, stream.Channel_Blue, stream.Channel_Alpha}
 }
+
+// ASTCFrom reads a raw astc image, extracts the header and creates an
```

Comment:
Mention here that it is expecting the image with a header, which is a format that is only used when storing the images in our test data files. I think the astc_encoder CLI tool can output images in this format.

Reply:
Yes it can output with this format. 

Currently I am returning the images without the header simply because I don't know how we will use the images yet. If we require to have the header, I can add the header as well that's not a big issue. My guess is we will not need as we already have the header information when compressing but if we need, it's not hard to add.